### DISCUSSION TITLE
Is v1.30.1 still stable/usable?

### DISCUSSION THREAD

**Discussion by spazer5** [socketry/async#160] [Q&A] [answered]
I'm on a windows machine and unable to install v2 due to `io-event` gem not building right.

I tried v1 and it looks to be working perfectly, so I'm wondering if it's still a stable version for a ruby 3.1.1 app.

**Comment by ioquatix** [comment, marked as answer]
Yes, Async 1.x is the most stable and will be supported for the foreseeable future. Async 2.x is a superset and will be compatible with Async 1.x. The difference is, Async 2.x has more opportunities for concurrency.
